code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -27,10 +28,58 @@ var/const/MAX_ACTIVE_TIME = 400
 	var/strength = 5
 
 	var/attached = 0
+	var/target_time = 0.5 // seconds
+	var/walk_speed = 2 // tenths of seconds
```

Comment:
Actually I'll change that to a lower value anyway, facehuggers seem to be less active than I'd like